site/app/controllers/admin/ConfigurationController.php

```diff
@@ -150,6 +153,18 @@ public function updateConfiguration(): MultiResponse {
         ) {
             $entry = $entry === "true" ? true : false;
         }
+        elseif ($name === 'queue_enabled') {
+            $entry = $entry === "true" ? true : false;
+        }
+        elseif ($name === 'queue_contact_info') {
+            $entry = $entry === "true" ? true : false;
+        }
+        elseif ($name === 'polls_enabled') {
+            $entry = $entry === "true" ? true : false;
+        }
+        elseif ($name === 'upload_message') {
+            $entry = nl2br($entry);
+        }
```

Comment:
Did you mean to change 165-167. I am not 100% sure but I think this has nothing to do with polls. If it does, just ignore me